python/_tskitmodule.c

```diff
@@ -10760,7 +10760,7 @@ TreeSequence_get_individuals_metadata_offset(TreeSequence *self, void *closure)
     }
     individuals = self->tree_sequence->tables->individuals;
     ret = TreeSequence_make_array(
-        self, individuals.num_rows + 1, NPY_UINT32, individuals.metadata_offset);
+        self, individuals.num_rows + 1, NPY_UINT64, individuals.metadata_offset);
```

Comment:
Hmm, did the tests pass before because we didn't look at the contents of the offset arrays?

Reply:
Yeah, the low-level tests don't check the contents, just the semantics and lifetime/memory concerns.